.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: csharp
+solution: Banshee.sln
+
+addons:
+  apt:
+    packages:
+      - libgtk3.0-cil
+      - libdbus2.0-cil
+      - libtaglib2.1-cil
+      - libmono-addins0.2-cil
+      - libdbus-glib2.0-cil
+      - libgudev3.0-cil
+      - libmono-zeroconf1.0-cil
+      - libgconf2.0-cil
+env:
+  - BUILD_CONFIGURATION=Release
+  - BUILD_CONFIGURATION=Debug
+
+script:
+  - msbuild /p:Configuration=$BUILD_CONFIGURATION Banshee.sln
```

Comment:
Makes sense, both systems should be able to build and we'll be building a full blown package using this configuration rather than a subset - are there any solution or project file changes needed?

I'm pretty sure autotools works on Windows too though :P